R/pkg/R/functions.R

```diff
@@ -2260,6 +2293,32 @@ setMethod("from_csv", signature(x = "Column", schema = "characterOrColumn"),
             column(jc)
           })
 
+#' @details
+#' \code{schema_of_csv}: Parses a CSV string and infers its schema in DDL format.
+#'
+#' @rdname column_collection_functions
+#' @aliases schema_of_csv schema_of_csv,characterOrColumn-method
+#' @examples
+#'
+#' \dontrun{
+#' csv <- "'Amsterdam,2018'"
```

Comment:
I"m a bit confused `"'Amsterdam,2018'"` vs `"Amsterdam,2018"`
does the latter work?